### PR TITLE
Bind static analysis checks to 'verify'

### PR DESCRIPTION
**Pull request by seanf**
See also https://github.com/zanata/zanata-parent/pull/9 and https://github.com/zanata/zanata-parent/pull/12

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-parent/60)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -256,16 +256,15 @@
...
-        <!--
-                <executions>
-                  <execution>
-                    <goals>
-                      <goal>check</goal>
-                    </goals>
-                    <phase>verify</phase>
-                  </execution>
-                </executions>
-        -->
```
Comment (aeng) [inline pom.xml]: @seanf Do we know why it was commented out?

**Comment by aeng** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-parent/60)*

**Comment by seanf** [comment]
From what I can tell, the lifecycle 'verify' binding was already commented out when checkstyle was first added (probably because we had to fix the style in all Zanata code before we could enforce it). It was only activated by command-line parameters which were provided by Jenkins (checkstyle:check). The intention was to uncomment it once the style was cleaned up, but this never happened.

At one point, @huangp integrated static analysis into the 'test-compile' lifecycle phase, but this caused problems compiling zanata-server and had to be backed out.

<details><summary><i>Previously, aeng (Alex Eng) wrote…</i></summary>
> *(Reviewable was [unable](https://github.com/Reviewable/Reviewable/issues/14) to map this GitHub line comment to the right spot--sorry!)*
> 
> @seanf Do we know why it was commented out?
</details>


---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-parent/60#-:-KIWAwTWPRXcu5CATZ5F:-174177144)*

**Comment by seanf** [comment]
@huangp Do you want to take a look at this too?

**Comment by huangp** [comment]
:lgtm:

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-parent/60#-:-KM7bTp7LXnyvo6ZUHBV:1417088721)*

**Pull request merged**